src/style.css

```diff
@@ -1 +1,61 @@
-/* Styles go here */
+body {
+  margin: 0;
+  font-family: Roboto, sans-serif;
+  font-size: 12px;
+  font-weight: 500;
+  --color-link: #00acdc;
+}
+
+.nav__list {
+  list-style: none;
+  padding: 0;
+  margin: 0;
+  display: flex;
+}
+
+.header {
+  padding-left: 50px;
+  padding-right: 50px;
+  height: 60px;
+  text-transform: uppercase;
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+}
+
+.logo {
+  margin-right: 20px;
+  display: flex;
+}
+
+.nav_item {
+  position: relative;
+}
+
+.nav_item a {
+  align-items: center;
+  padding: 23 0;
```

Comment:
```suggestion
  padding: 23px 0;
```
![image](https://user-images.githubusercontent.com/86923089/216814056-05fe5ca7-ebea-486a-a4a6-62d28bf6455b.png)